src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalAmount = days * 40;
+  let amount = 0;
+  const d7 = 50;
```

Comment:
create more meaningful names for variables

Reply:
Hope those are legible